*code revised and shortened  *profileViewRight & profileViewLeft is needed for the alignment of myProfile

MoMa666 opened the pull request:
…in content.css,

*style code commented out in classes and inserted into css
*layoutstyle of header edited and of logout icon
*flexoption wrap applied to content so that the objects are displayed below each other as soon as the window is reduced (example can be seen in myProfile)
*sideBar no longer overlaps with the content, this was also changed by reworking the css code.

MoMa666 committed:
*add background, fontart and color for vertical and horizontal views in content.css,
*style code commented out in classes and inserted into css
*layoutstyle of header edited and of logout icon
*flexoption wrap applied to content so that the objects are displayed below each other as soon as the window is reduced (example can be seen in myProfile)
*sideBar no longer overlaps with the content, this was also changed by reworking the css code.

sabrine-gamdou commented on frontend/styles/views/main/mainView.css:
```diff
@@ -39,22 +40,25 @@
 
 #logout{
   width: auto;
-  height: 60px;
```
Besser ist es mit 2.8rem=45px zu arbeiten.

sabrine-gamdou commented on frontend/styles/views/main/mainView.css:
```diff
@@ -39,22 +40,25 @@
   width: auto;
-  height: 60px;
+  height: 45px;
+  left: 7%;
```
left erzeugt bei mir einen weißen Rand rechts, ohne passt es sich an.
Ohne left ist es besser!

sabrine-gamdou commented on frontend/styles/views/main/content.css:
```diff
@@ -1,9 +1,123 @@
+ *@since 07.01.2021
+ */
+
+#content-view_blue {
```
Bitte in SDP schauen, Variablen nach der dort benannten Konvention benennen! Hier "contentViewBlue"

VannaSky commented on frontend/styles/views/main/content.css:
```diff
@@ -1,9 +1,123 @@
+    height: 478px;
+    left: 7%;
 }
 
```
Bitte die .pageContentPositon-Klasse aus mainView.css in content.css kopieren und alle Attribute, die auf allen Seiten gesetzt werden, dort zusammen fassen. Das spart viele Zeilen Code ein :-)

VannaSky commented on frontend/styles/views/main/content.css:
```diff
@@ -1,9 +1,123 @@
+ */
+
+#content-view_blue {
+    width: 80%;
+    height: 315%;
```
zu .pageContentPosition

MoMa666 committed:
*code revised and shortened
*profileViewRight & profileViewLeft is needed for the alignment of myProfile

MoMa666 committed:
*content is now always inserted in the mainView and no longer in contentHolder
*contentHolder is no longer needed
*the routes have been changed from contentHolder to mainView
*the content is in a horizontal layout together with the sidebar. Therefore a vertical layout was created so that it can be displayed in the middle of the view.
*The content is now correctly connected to a layout.
*Css classes were added according to application areas and the css were ordered accordingly.
*A header class was created to move the code out of the mainView.

MoMa666 commented:
make new request!
